libs/frontend/game/src/lib/+state/facade/facade.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { BoardMap } from '../reducers';
+import { getBoard } from '../selectors/selectors';
+import { BoardActions } from '../actions/actions';
+import { DataCell } from '../../board/table';
+
+@Injectable({ providedIn: 'root' })
+export class BoardFacade {
+  board$ = this.store.select(getBoard);
+
+  constructor(private store: Store<BoardMap>) {
+  }
+
+  selectCell(data: DataCell) {
+    this.store.dispatch(BoardActions.markCellAsActive({ data }));
+  }
+
+  unSelectCell(data: DataCell) {
+    this.store.dispatch(BoardActions.markCellAsUnactive({ data }));
+
+  }
+}
```

Comment:
a nie mielismy robic klas od interfejsów? Tak jak zrobiłeś te sockety

Reply:
jasne sluszna uwaga, aczkolwiek ta struktura to nie serwis, to jest fasada ktorej zadaniem wystawienie tylko prostych funkcji ktore moga byc uzywane z komponentow. Fasady nigdy nie podmienimy na inna klase tego sameto typu z racji tego ze i sam state nie ulega podmianie